payjoin/src/core/receive/common/mod.rs

```diff
@@ -144,6 +144,9 @@ impl WantsOutputs {
         })
     }
 
+    #[cfg(feature = "v2")]
+    pub(crate) fn payjoin_proposal(&self) -> Psbt { self.payjoin_psbt.clone() }
```

Comment:
I see why you preferred the field to this in #1102 because the feature flag can be avoided. Though I'm not sure why you're using pub(crate) when pub(super) is sufficient.